config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+
+development:
+  <<: *default
+  database: db/development.sqlite3
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: db/test.sqlite3
+
+production:
```

Comment:
Config this file into .gitignore then run bellow command line --> do not push this file to pull request
git rm --cached config/database.yml
Create a new file with name "config/database.yml.example" --> this file using push to pull request

Reply:
not fix yet

Reply:
this file will not be up to GIt, please
cp database.yml database.yml.example
git rm --cached config/database.yml
and in .gitignore add
config/database.yml